client/sites/components/dotcom-style.scss

```diff
@@ -290,10 +290,21 @@
 		ul.dataviews-view-list {
 			li.is-selected,
 			li.is-selected:hover {
-				background-color: #ebf2fc;
+				// Override this element to ensure it doesn’t blend with the Core's alpha-based color.
```

Comment:
These styles are targeting internal dataviews classes, like `ul.dataviews-view-list`. This is fragile making updates harder.

I don't know why we need these styles, can we remove them? If we can't for now, and we want to add them temporarily to be removed later, my suggestion is to 1) add them to [this file](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/dataviews/style.scss#L192) and 2) provide a link to an issue/pr where it's described why it's necessary and the plan for removal.